src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -776,6 +831,78 @@ static int Lambda(int x, Func<int, int> l)
 True
 True
 True
+True
+";
+
+            CompileAndVerify(source + InstrumentationHelperSource, expectedOutput: expectedOutput, options: TestOptions.ReleaseExe);
+            CompileAndVerify(source + InstrumentationHelperSource, expectedOutput: expectedOutput, options: TestOptions.DebugExe);
+        }
+
+        [Fact]
+        public void MultipleFilesCoverage()
```

Comment:
Taking another look, I'm not seeing any fake indices less than 1, so "SourceDocument 1/2/3/4" would be expected.